src/Razor/Microsoft.NET.Sdk.Razor/test/GenerateBlazorBootJsonTest.cs

```diff
@@ -7,29 +7,28 @@
 using Microsoft.Build.Framework;
 using Moq;
 using Xunit;
-using BootJsonData = Microsoft.AspNetCore.Components.WebAssembly.Build.GenerateBlazorBootJson.BootJsonData;
 
-namespace Microsoft.AspNetCore.Components.WebAssembly.Build
+namespace Microsoft.AspNetCore.Razor.Tasks
 {
-    public class GenerateBlazorBootJsonTest
+    public class GenerateBlazorWebAssemblyBootJsonTest
     {
-        [Fact]
+        [Fact(Skip = "TODO fix this")]
```

Comment:
Not sure if you're planning to handle this as part of the PR or later. If later, could you add it to the list of things to be tracked for follow-up? Presumably we could have one big issue listing them all.